scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -849,23 +904,23 @@ function Get-AgentRegistry {
     return 'index.docker.io'
 }
 
-function Stop-IotEdgeContainers {
+function Remove-IotEdgeContainers {
     $dockerExe = Get-DockerExePath
 
     if (-not (Test-IsDockerRunning 6> $null)) {
         return
     }
 
-    $runningContainersString = Invoke-Native "$dockerExe ps --format ""{{.ID}}""" -Passthru
+    $runningContainersString = Invoke-Native "$dockerExe ps --all --format ""{{.ID}}""" -Passthru
```

Comment:
Thanks...I noticed this bug in the last couple days, but didn't know the cause...

Reply:
The bug is the double `ConvertFrom-JSON` actually.

Reply:
Ah, the change a couple lines down...